lib/improver/wxcode/wxcode_utilities.py

```diff
@@ -81,9 +81,9 @@ def add_wxcode_metadata(cube):
     cube.standard_name = None
     cube.var_name = None
     cube.units = "1"
-    wx_keys = np.array(WX_DICT.keys())
+    wx_keys = np.array(list(WX_DICT.keys()))
```

Comment:
Not sure about these lists

Reply:
I think this list is actually needed, as otherwise the attribute is:
```python
array(odict_keys([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30]), dtype=object)
```
rather than 
```python
array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16,
       17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30])
```